src/v2/controllers/vreplicaset_controller/model/reconciler.rs

```diff
@@ -97,7 +97,7 @@ pub open spec fn reconcile_core(v_replica_set: VReplicaSetView, resp_o: Option<R
                                 obj: pod.marshal(),
                             });
                             let state_prime = VReplicaSetReconcileState {
-                                reconcile_step: VReplicaSetReconcileStep::AfterCreatePod((diff - 1) as usize),
+                                reconcile_step: VReplicaSetRecStepView::AfterCreatePod(abs(diff - 1)),
```

Comment:
Why do we need `abs` here? Is it to make sure the result is still a `nat` since there is subtraction here?

Reply:
The compiler complains if the expression is left as-is without a cast/operation. If you'd prefer `as nat`, I can do that change.